consensus/src/epoch_manager.rs

```diff
@@ -306,22 +306,13 @@ impl EpochManager {
             MetricsSafetyRules::new(self.safety_rules_manager.client(), self.storage.clone());
         match safety_rules.perform_initialize() {
             Ok(()) => { /* Expected */ }
-            Err(Error::ValidatorKeyNotFound(error)) => {
+            Err(error) => {
```

Comment:
probably could have just done a if let Err(e) = safety_rules.perform_initialize() {

oh well

Reply:
in some ensuing PR we should probably add a counter that stores the LSR current state (enabled or disabled)

Reply:
> probably could have just done a if let Err(e) = safety_rules.perform_initialize() {

Hah! This is what happen when I refactor too quickly :P

> in some ensuing PR we should probably add a counter that stores the LSR current state (enabled or disabled)

I agree; we can add this to the list of things we should do/fix when we tackle the observer mode. For now, the ugliness is just delegated to the runbook 😄